src/components/WalletMenu.vue

```diff
@@ -118,7 +118,7 @@ export default class WalletMenu extends Vue {
 
     @Emit()
     // tslint:disable-next-line no-empty
-    private changePassphraseWallet(walletId: string) {}
+    private changePasswordWallet(walletId: string) {}
```

Comment:
This will then require a change in VWalletMenu in elements as well, because this changes the event name.